COPYRIGHT

```diff
@@ -0,0 +1 @@
+Copyright 2018 University of Washington
```

Comment:
Is this the new standard? Perhaps I haven't been keeping up, but I thought we put in an LSST copyright.

Reply:
https://developer.lsst.io/legal/copyright-overview.html

Reply:
I have no desire to claim copyright for UW but as far as I can see I'm just following the developer's guide. If there's a standard way to say "LSST" then I'll be happy to do that.

Reply:
UW is the correct copyright. There is no "LSST". There is LSST Corp (who aren't paying for this work) and there is NSF paying money through AURA to UW.